taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -1,62 +1,62 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
 import {
   Typography,
   Box,
   Grid,
   TextField,
-  Stack,
-  Autocomplete,
   Snackbar,
   Alert,
   CircularProgress,
   Button,
+  InputLabel,
+  FormControl,
+  Select,
+  MenuItem,
 } from "@mui/material";
-import { useState } from "react";
-import { useNavigate } from "react-router-dom";
+
 import { TAXONOMY_NAMES } from "../../constants";
 import { createBaseURL } from "../editentry/createURL";
-import { toSnakeCase } from "../../components/interConvertNames";
+import { toSnakeCase } from "../../utils";
 
 const StartProject = () => {
   const [branchName, setBranchName] = useState("");
-  const [taxonomyName, setTaxonomyName] = useState(null);
+  const [taxonomyName, setTaxonomyName] = useState("");
   const [description, setDescription] = useState("");
   const [loading, setLoading] = useState(false);
-  const [errorMessage, setErrorMessage] = useState(null);
+  const [errorMessage, setErrorMessage] = useState("");
   const navigate = useNavigate();
 
   const handleSubmit = () => {
-    const baseUrl = createBaseURL(taxonomyName, branchName);
+    if (!taxonomyName || !branchName) return;
```

Comment:
Might be just a personal prefrence, but 
```suggestion
    if (!(taxonomyName && branchName)) return;
```

Reply:
I'd prefer to keep it as is (return if any of those two is missing)